### ISSUE TITLE
GitHub workflow schema does not support input values for workflow_dispatch events

### ISSUE THREAD

**Issue by agateau-gg** [python-jsonschema/check-jsonschema#179]
I want to use `workflow_dispatch` events with input values in a GitHub workflow.

Here is an example:

```yaml
name: Workflow dispatch test

on:
  workflow_dispatch:
    inputs:
      name:
        description: "Name to greet"

jobs:
  hello:
    name: Hello
    runs-on: ubuntu-latest
    steps:
      - name: Say hello, name=${{ github.event.inputs.name }}
        run: |
          echo "Hello ${{ github.event.inputs.name }}!"
```

I configured pre-commit like this:

```yaml
default_stages: [commit]
repos:
  - repo: https://github.com/python-jsonschema/check-jsonschema
    rev: 0.18.4
    hooks:
      - id: check-github-workflows
```

Unfortunately the check fails, with an error I do not understand:

```
$ git commit
Validate GitHub Workflows................................................Failed
- hook id: check-github-workflows
- exit code: 1

Schema validation errors were encountered.
  .github/workflows/dispatch.yml::$.on: {'workflow_dispatch': {'inputs': {'name': {'description': 'Name to greet'}}}} is not valid under any of the given schemas
  Underlying errors caused this.
  Best Match:
    $.on: {'workflow_dispatch': {'inputs': {'name': {'description': 'Name to greet'}}}} is not of type 'string'
```

I bypassed the check and pushed: the workflow works on GitHub.

You can find the code in this example repository: https://github.com/agateau-gg/workflow-dispatch-test, and an example of a run: https://github.com/agateau-gg/workflow-dispatch-test/actions/runs/3436134455/jobs/5729305488.

**Comment by sirosen** [comment]
Thanks for reporting this!

I've been struggling to make time for check-jsonschema the past few weeks, so there hasn't been a release for a while. Based on a quick test I did, the schema (which comes from schemastore) already supports this and I just need to release. I'll try to polish off the work I started on and push a release out today.

**Comment by sirosen** [comment]
This should have been resolved in v0.19.0, just released.
Please let me know if you continue to see this issue, or if you run into any new problems.

Once again; thanks for reporting!

**Comment by agateau-gg** [comment]
I confirm it works now. Thanks a lot for the quick fix!
